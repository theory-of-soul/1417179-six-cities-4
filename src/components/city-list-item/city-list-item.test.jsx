import React from "react";
import renderer from 'react-test-renderer';
import CityListItem from "./city-list-item";

const city = `Amsterdam`;

describe(`CityListItem component snapshot tests`, () => {
  it(`CityListItem component is active`, () => {
    const tree = renderer
      .create(<CityListItem isActive={true} city={city} onCityClickHandler={() => {}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`CityListItem component is inactive`, () => {
    const tree = renderer
      .create(<CityListItem isActive={false} city={city} onCityClickHandler={() => {}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
