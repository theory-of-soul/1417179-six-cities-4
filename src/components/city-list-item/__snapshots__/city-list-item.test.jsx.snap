// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityListItem component snapshot tests CityListItem component is active 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;

exports[`CityListItem component snapshot tests CityListItem component is inactive 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item"
    href="#"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
